typing/typedtreeIter.ml

```diff
@@ -249,7 +250,7 @@ module MakeIterator(Iter : IteratorArgument) : sig
         exp.exp_extra;
       begin
         match exp.exp_desc with
-          Texp_ident (path, _, _) -> ()
+        | Texp_ident (path, _, _) -> ()
```

Comment:
If anything, I would complain about the *absence* of the first `|`. But I've learned not to complain too much.